### PR TITLE
add bors and change github action

### PR DESCRIPTION
**Pull request by bidoubiwa**
Waiting on #551

### CODE REVIEW

**Comment by bidoubiwa** [comment]
bors try

**Review by curquiza — DISMISSED** [review]

**Comment by bidoubiwa** [comment]
bors merge

File: .github/workflows/check-links.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
 ---
 name: Check dead links
-on: [pull_request]
+on:
+  pull_request:
+  push:
+    branches:
+      - trying
+      - staging
```
Comment (curquiza) [inline .github/workflows/check-links.yml:8]:
```suggestion
      - staging
      - master
```

File: .github/workflows/check-links.yml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 ---
 name: Check dead links
-on: [pull_request]
+on:
+  pull_request:
+  push:
```
Comment (curquiza) [inline .github/workflows/check-links.yml:5]:
```suggestion
  # trying and staging branches are for BORS config
  push:
```

File: .github/workflows/check-style.yml [hunk 3]
```diff
@@ -1,8 +1,14 @@
 ---
 name: Check styling
-on: [pull_request]
+on:
+  pull_request:
+  push:
```
Comment (curquiza) [inline .github/workflows/check-style.yml:5]:
```suggestion
  # trying and staging branches are for BORS config
  push:
```

File: .github/workflows/check-style.yml [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
+    branches:
+      - trying
+      - staging
```
Comment (curquiza) [inline .github/workflows/check-style.yml:8]:
```suggestion
      - staging
      - master
```

File: .github/workflows/gh-pages-scraping.yml [hunk 5]
```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   build-deploy:
+    name: build github pages
```
Comment (curquiza) [inline .github/workflows/gh-pages-scraping.yml:10]: No `push` event configuration in this file?

```yml
on:
  pull_request:
  push:
    # trying and staging branches are for BORS config
    branches:
      - trying
      - staging
      - master
```

**Review by curquiza — CHANGES_REQUESTED** [review]

**Comment by bidoubiwa** [comment]
bors try

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
Will work once this issue is fixed: https://github.com/meilisearch/documentation/issues/551

**Comment by fharper** [comment]
@bidoubiwa I changed the status to draft since we are waiting for a bug to be fixed.

**Comment by react-learner** [comment]
🚨 Blocked by #551 🚨

**Comment by curquiza** [comment]
This PR is replaced #977 
#551 is not blocking to add bors but is still an issue to fix 😇

**Pull request closed without merging**